Explorer/Assets/DCL/Chat/ChatController.cs

```diff
@@ -460,7 +460,7 @@ private void CreateChatEntry(ChatMessage chatMessage)
                 world.AddOrGet(entity, new ChatBubbleComponent(chatMessage.Message, chatMessage.Sender, chatMessage.WalletAddress));
                 UIAudioEventsBus.Instance.SendPlayAudioEvent(viewInstance!.ChatReceiveMessageAudio);
             }
-            else if (chatMessage.SystemMessage == false)
+            else if (chatMessage is {SystemMessage: false, SentByOwnUser: true })
```

Comment:
What if we clean this a little bit to make it easier to read? Something like
```c#
if(chatMessage.SentByOwnUser){
   if(chatMessage == false){ ...}
else{
   if(entityParticipantTable.Has(....)) {...}
   }
``` 

What do you think?